osu.Game/Tests/Visual/ScreenTestScene.cs

```diff
@@ -29,8 +29,9 @@ protected ScreenTestScene()
 
         protected void LoadScreen(OsuScreen screen)
         {
-            if (Stack.CurrentScreen != null)
+            while (Stack.CurrentScreen != null)
                 Stack.Exit();
```

Comment:
I'm pretty sure `TestScenePause` is causing this to enter an infinite loop due to `ValidForResume = true` at the end of some tests, which makes `Exit()` not unwind the stack.

Reply:
good catch